### ISSUE TITLE
PlayRunHook's afterStopped method doesn't run on Windows

### ISSUE THREAD

**Issue by arpsmack** [playframework/playframework#11267]
### Play Version
2.8.13

### API
Java

### Operating System
Windows 10

### JDK
openjdk version "11.0.13" 2021-10-19 LTS
OpenJDK Runtime Environment Zulu11.52+13-CA (build 11.0.13+8-LTS)
OpenJDK 64-Bit Server VM Zulu11.52+13-CA (build 11.0.13+8-LTS, mixed mode)

### Expected Behavior
1. Install a PlayRunHook with an afterStopped method
2. Run your application in development
3. Stop your application (I usually stop it by clicking the "Stop '$ProjectName'" button in IntelliJ)
4. The afterStopped method is called when you stop your application

### Actual Behavior
1. The afterStopped method is never run

### Reproducible Test Case
I can make one if necessary I just don't have time right now. Hopefully the problem is simple enough that it isn't needed.

### More context
This is the only other thing I could find searching for this issue: https://stackoverflow.com/questions/47576368/playrunhook-afterstopped-method-not-being-called-on-windows

We use a run hook to run a `webpack --watch` and kill the corresponding process in the afterStopped method. Our developers that use macs have no problems with this, but on Windows the webpack watchers continue running. We often forget about it, and after working on the project for a couple hours, we realize we have 20+ node processes all running webpack watchers.

**Comment by mkurz** [comment]
Does this problem also happen outside of IntelliJ?

**Comment by arpsmack** [comment]
I added some logging to our hook and ran SBT manually from the command line, and sure enough both exiting gracefully via the Enter key, and exiting via `CTRL+C` cause the afterStopped hook to run and log, but the node processes never get killed.

It still doesn't log when I exit via the stop button in IntelliJ, but perhaps IntelliJ is detaching the output before the hooks run. I just tested this by having the hook write out a file when it stops, and the file doesn't get written out when stopping via IntelliJ, so it must be forcefully terminating the SBT process without allowing the hooks to run (bummer, not sure how to work around that).

Since the node processes still aren't getting killed even when the hook does run, I'd say it's really likely that there's also a bug on our end (probably something to do with process parent-child relationship differences between unixy systems and Windows).

Either way, it appears I was wrong and this probably isn't a bug in the Play hooks.
